### PR TITLE
update sqlfluff version in gettingstareted.rst

### PR DESCRIPTION
**Pull request by keyem4251**
update sqlfluff version to 1.2.1 from 0.3.1 in gettingstaretd.rst

### CODE REVIEW

File: docs/source/gettingstarted.rst [hunk 1]
```diff
@@ -61,7 +61,7 @@ version number.
 .. code-block:: text
 
     $ sqlfluff version
-    0.3.1
+    1.2.1
```
Comment (barrywhart) [inline docs/source/gettingstarted.rst:64]: @tunetheweb, @alanmcruickshank: How difficult would it be to automatically generate/update this as part of the release process?

Comment (tunetheweb) [reply docs/source/gettingstarted.rst:64]: @greg-finley ?

Comment (greg-finley) [reply docs/source/gettingstarted.rst:64]: Looks easy, I can send it over in a few hours.

**Review by barrywhart — APPROVED** [review]
This change looks good.

Sounds like @greg-finley will be automating this soon, so it doesn't get out of sync again. 🥳

**Comment by keyem4251** [comment]
Thank you for confirming!

**Pull request merged**